vtex-apps/store-icons
# Fix svg attributes in camel case

#### What is the purpose of this pull request?
Fixes all attributes with camel case in the `iconpack.svg` file. I also moved the `style` for the `sti-loading` to a global css file imported in the `Icon` component.

#### What problem is this solving?
These attributes are not HTML spec compliant, so pages (or a really old browser) that require strict HTML validation won't load correctly. One use case suffering from this is the AMP store, because the amp-validator requires a "perfect" HTML page.

The reason for the extraction of the `style` on the `sti-loading` is also because of the amp validator. In a AMP page, it's required for *all* css to be in a single `style[amp-custom]` tag in the head, so moving that style tag to the css file imported in the component I'm able to render that into the document, but in the appropriate place. This change doesn't affect the normal store, because the style will also be added to the head when you import the component.

#### How should this be manually tested?
[Workspace](https://lucas--storecomponents.myvtex.com/amp/). It's a good idea to use the [amp extension](https://chrome.google.com/webstore/detail/nmoffdblmcmgeicmolmhobpoocbbmknc) to actually see the errors, or the absence of them if the icon is green.

#### Types of changes
